lang/en.json

```
@@ -1647,6 +1647,13 @@
 "SETTINGS.5eAutoCollapseCardN": "Collapse Item Cards in Chat",
 "SETTINGS.5eAutoSpellTemplateL": "When a spell is cast, defaults to begin the process to create the corresponding Measured Template if any (requires TRUSTED or higher player role)",
 "SETTINGS.5eAutoSpellTemplateN": "Always place Spell Template",
+"SETTINGS.5eAttackRollCheckVisibility": {
+  "Name": "Show attack roll result to players",
+  "Hint": "Control visibility of AC check cards.",
```

Just to make the name match closer to other settings we can move more information into the hint:

```suggestion
  "Name": "Attack Result Visibility",
  "Hint": "Control visibility of attack roll results in chat cards for players.",
```